ci/setup.sh

```diff
@@ -1,10 +1,23 @@
 #!/bin/bash -eu
 
-if [ "$TRAVIS_OS_NAME" == "linux" ]; then
-  sudo modprobe fuse
+: ${INTEGRATION:=false}
+: ${CHECK:=false}
+
+if [[ "$INTEGRATION" == "true" ]]; then
+  if uname -s | grep -q Linux; then
+    sudo modprobe fuse
+  elif uname -s | grep -q Darwin; then
+    brew cask install osxfuse
+  fi
 fi
 
-if [ "$TRAVIS_OS_NAME" == "osx" ]; then
-  brew cask install osxfuse
+if [[ "$CHECK" == "true" ]]; then
+  if uname -s | grep -q Linux; then
+    wget https://cmake.org/files/v3.9/cmake-3.9.1-Linux-x86_64.tar.gz -O /tmp/cmake.tar.gz
```

Comment:
hmm, since this script is only used w/ CI, I think we should just remove this when/if the travis image we use comes with a later cmake.